ipysheet/easy.py

```diff
@@ -67,8 +67,7 @@ def current():
 
 
 def cell(row, column, value=0., type=None, color=None, background_color=None,
-         font_style=None, font_weight=None, style=None, label_left=None, choice=None,
-         read_only=False, format='0.[000]', renderer=None):
```

Comment:
Why is the line with read_only removed?

Reply:
It's not used explicitly in the function, just passed to the `Cell` constructor. It's a keyword argument so after this change it will be given to `**kwargs` and forwarded to the `Cell` constructor with `**kwargs`. It's basically the same code but cleaner/easier to read/maintain.

Reply:
They are there for tab completion, easier to maintain maybe, but this is mainly for usage :) Could you restore it? Every trait should be an argument here.

Reply:
The more we will add features, the more we will need to add new parameters to this function... Here I would have to add the `numeric_format` and the `date_format`. I'm not sure I like this. Having 15 parameters to a function does not feel right.